mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule Quantum.Mixfile do
 
   use Mix.Project
 
-  @version "3.2.0"
+  @version "3.3.0"
```

Comment:
Please remove that line sice the release will be a separate commit.

Reply:
Done :)